pkgs/applications/networking/remote/teamviewer/default.nix

```diff
@@ -21,19 +39,36 @@ mkDerivation rec {
   '';
 
   nativeBuildInputs = [ autoPatchelfHook makeWrapper wrapQtAppsHook ];
-  buildInputs = [ dbus getconf qtbase qtwebkit qtwebengine qtx11extras libX11 ];
-  propagatedBuildInputs = [ qtquickcontrols ];
+  buildInputs = [ dbus getconf qtbase qtwebengine qtx11extras libX11 icu63 ];
```

Comment:
Are those being picked up by autopatchelfHook? If they are only used for LD_LIBRARY_PATH then this is a noop.

Reply:
good point. Only some are used for autopatchelf. I removed the unnecessary ones